index.html

```diff
@@ -0,0 +1,58 @@
+<!DOCTYPE html>
+<html lang="en">
+
+<head>
+  <meta charset="UTF-8" />
+  <meta http-equiv="X-UA-Compatible" content="IE=edge" />
+  <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+  <title>Portifolio skeleton</title>
+  <link rel="stylesheet" href="index.css" />
+</head>
+
+<body>
+  <div class="font">
+    <section class="first-section">
+      <div class="first-container">
+        <div class="font">
+          <header class="header">
+            <a class="my-logo">My Logo</a>
+            <div class="hamburger">
+              <ul class="ul-style">
+                <li>
+                  <a href="#">
+                    <img src="./images/Union.svg" alt="union background" />
+                  </a>
+                </li>
+              </ul>
```

Comment:
- [ ] Kindly make sure that the website is responsive on the mobile version
<img width="390" alt="Screen Shot 2022-04-22 at 13 38 52" src="https://user-images.githubusercontent.com/28307960/164708599-d752ef76-a74f-46af-bbc5-4f4bfe08ae81.png">